Pass the configured path options to the build details generator.

--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1002,5 +1002,5 @@
 
 $(BUILD_DETAILS): pybuilddir.txt
-	$(RUNSHARED) $(PYTHON_FOR_BUILD) $(srcdir)/Tools/build/generate-build-details.py `cat pybuilddir.txt`/$(BUILD_DETAILS)
+	$(RUNSHARED) $(PYTHON_FOR_BUILD) $(srcdir)/Tools/build/generate-build-details.py $(BUILD_DETAILS_FLAGS) `cat pybuilddir.txt`/$(BUILD_DETAILS)
 
 # Build static library
